libimage/pull.go

```diff
@@ -487,14 +490,12 @@ func (r *Runtime) copySingleImageFromRegistry(ctx context.Context, imageName str
 	if options.OS != runtime.GOOS {
 		lookupImageOptions.OS = options.OS
 	}
+
 	// FIXME: We sometimes return resolvedImageName from this function.
 	// The function documentation says this returns an image ID, resolvedImageName is frequently not an image ID.
 	//
 	// Ultimately Runtime.Pull looks up the returned name... again, possibly finding some other match
 	// than we did.
```

Comment:
That return value is now gone, so this comment can really be deleted, after all.